packages/chat-component-bindings/src/utils/constants.ts

```diff
@@ -3,3 +3,12 @@
 
 export const MINIMUM_TYPING_INTERVAL_IN_MILLISECONDS = 8000;
 export const PARTICIPANTS_THRESHOLD = 20;
+
+export const ACSKnownMessageType = {
```

Comment:
This should be fixed here to not require this enum: https://github.com/Azure/azure-sdk-for-js/issues/15276
(might not be released yet though?)
But doesn't look to suppoer richtext/html https://github.com/Azure/azure-sdk-for-js/blob/12002a4ae839df764ba6415e8ee2d0bc6b9a4858/sdk/communication/communication-chat/src/generated/src/models/index.ts#L301

Reply:
the html/richtext is coming from teams

I think we need to support text, html and html/richtext

Reply:
Does an issue need filed on the chat team to add `html/richtext` to this enum?

Reply:
Or perhaps they need a new enum `enum ChatMessageTeamsInteropType = ChatMessageType | 'html/richtext'` or something. I think we can definitely keep this in for now but can you file an issue on the chat team and add a comment here we should remove this type once the chat team has a fix/supported type we can use?

Reply:
Looks like this is not in released version of js sdk, lemme leave a TODO here and we could check it later when it comes